service/build.gradle

```diff
@@ -92,6 +92,10 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework.security:spring-security-core'
 
+    // JWT dependencies (used to parse access token)
```

Comment:
nit: "used to parse token from Auth0"?

Reply:
Updated, avoiding `for auth0` since this can be used for other token issuers as well. there are other third party libraries as well, though not widely used